docs/en/latest/plugins/request-validation.md

```diff
@@ -39,10 +39,11 @@ For more information on schema, refer to [JSON schema](https://github.com/api7/j
 
 ## Attributes
 
-| Name          | Type   | Requirement | Default | Valid | Description                |
-| ------------- | ------ | ----------- | ------- | ----- | -------------------------- |
-| header_schema | object | optional    |         |       | schema for the header data |
-| body_schema   | object | optional    |         |       | schema for the body data   |
+| Name             | Type   | Requirement | Default | Valid | Description                |
+| ---------------- | ------ | ----------- | ------- | ----- | -------------------------- |
+| header_schema    | object | optional    |         |       | schema for the header data |
+| body_schema      | object | optional    |         |       | schema for the body data   |
+| rejected_message | string | optional    |         |       | schema for the customize reject message |
```

Comment:
It's better to remind users that at least one of `header_schema` and `body_schema` should be specified.

Reply:
Roger that.